sdks/go/workflow.go

```diff
@@ -356,10 +364,10 @@ func (w *Workflow) NewTask(name string, fn any, options ...TaskOption) *Task {
 	fnType := fnValue.Type()
 
 	if fnType.Kind() != reflect.Func {
-		panic("task function must be a function")
+		panic("task input must be a function")
```

Comment:
same here I think 😅 - sorry! agree the naming is a bit confusing though. I'd usually say "`fn` must be a function" if we really need the `any`, but is it really not possible to not use `any` here? I'm a bit surprised by that